pkg/model/pki.go

```diff
@@ -34,6 +34,9 @@ type PKIModelBuilder struct {
 
 var _ fi.ModelBuilder = &PKIModelBuilder{}
 
+// keypairType is used to differentiate the old secretes with the newer versions
+const keypairType = "Keypair"
```

Comment:
What is the other type?  I think maybe we have a field Format or "APIVersion", and we have "legacy" & "kops/v1alpha2" as that most closely matches what we're actually doing I think.

Although the obvious problem with this is that I don't know how easy it is to get the stored version from the apimachinery.  But I'm OK with cheating at the moment (assuming v1alpha2)

Reply:
This was the only thing in the API object that I could find that was actually set.  I tested the heck out of this, and the only thing I could figure out how to access is the object name Keypair.